frontend/src/styles/element.scss

```diff
@@ -263,3 +263,10 @@ html {
     background-color: #f5f7fa !important;
     color: var(--el-color-info) !important;
 }
+
+.custom-input-textarea:hover {
+    color: var(--el-color-primary) !important;
+    background-color: var(--el-color-primary-light-9) !important;
+    border-color: var(--el-button-border-color) !important;
+}
+
```

Comment:
The provided CSS snippet appears to be intended for styling elements with the class `.custom-input-textarea`. Here's a review of the changes:

1. **Variable Usage**: The use of `var(--el-color-info)` is consistent and likely correctly referenced from an existing theme.

2. **Hover Styling Improvements**:
   - **Color Change**: Changing the hover state color from gray to primary blue (`#007bff` or similar) aligns well visually with typical form fields.
   - **Background Color**: A lightened version (`var(--el-color-primary-light-9)`) enhances readability, especially in dark themes.
   - **Border Color**: Ensuring the border remains aligned with other UI components provides unity and consistency across your application design.

These changes improve user experience on hover actions by providing immediate feedback without altering the overall interface significantly. Good job on enhancing the visual appeal and usability directly related to input text areas!